e2e_playwright/deploy_dialog_test.py

```diff
@@ -34,8 +34,12 @@ def test_deploy_button_displays_correctly(
         deploy_dialog.get_by_test_id("stDeployDialogCustomDeploymentIcon")
     ).to_be_visible()
 
-    # Wait for a short time to make sure that the images have been loaded
-    themed_app.wait_for_timeout(250)
+    # Wait to make sure that the images have been loaded
+    themed_app.wait_for_function("""() => {
```

Comment:
nit: this might be a nice utility for the shared utils methods.

Reply:
good idea, moved it to a `wait_for_all_images_to_be_loaded` function in `app_utils`.